### ISSUE TITLE
Crash while viewing 3D points - possibly as a new .copc finished converting

### ISSUE THREAD

**Issue by aritchie-usgs** [qgis/QGIS#56375]
### What is the bug or the crash?

Was viewing copcs (around 12 @ ~1-2GB/ea) in a 3D window panning and zooming. I had also added another .laz file that was converting to .copc. Right around the time that cloud was added to the view (and 3D view) My view zoomed waaay in (possibly user nav error) and everything froze up and the crash window opened. Not sure if the crash was from crazy fast zooming, or from the copc being added while zooming or the zooming was from the crash/copc being added to the live 3D view.

### Steps to reproduce the issue

Not sure if I can reproduce but here's a screenshot with the crash reporter on top of the screens as-crashed:

![20240115_q-crash](https://github.com/qgis/QGIS/assets/51339062/861c6987-26c1-406b-98dc-aaf7d5337bcf)

### Versions

QGIS version
3.34.3-Prizren
QGIS code revision
47373234ac
Qt version
5.15.3
Python version
3.9.18
GDAL/OGR version
3.8.3
PROJ version
9.3.1
EPSG Registry database version
v10.098 (2023-11-24)
GEOS version
3.12.1-CAPI-1.18.1
SQLite version
3.41.1
PDAL version
2.6.0
PostgreSQL client version
15.2
SpatiaLite version
5.1.0
QWT version
6.1.6
QScintilla2 version
2.13.4
OS version
Windows 10 Version 2009

Active Python plugins
mapswipetool_plugin
1.2
processing_saga_nextgen
1.0.0
profiletool
4.2.6
quick_map_services
0.19.33
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

## User Feedback

I was using the 3D view and accidentally zoomed way in then everything done froze. Right as one .laz finished converting into a .copc. not sure if that was related.

## Report Details

**Python Stack Trace**
```
Windows fatal exception: access violation

Thread 0x000026f0 (most recent call first):
<no Python frame>
```

**Stack Trace**

```

Qt3DCore::QChangeArbiter::sceneChangeEvent :
Qt3DRender::Render::Geometry::notifyExtentChanged :
Qt3DRender::Render::CalculateBoundingVolumeJob::run :
Qt3DCore::QThreadPooler::waitForAllJobs :
QThreadPoolPrivate::reset :
QThread::start :
BaseThreadInitThunk :
RtlUserThreadStart :
```

**QGIS Info**
QGIS Version: 3.34.3-Prizren
QGIS code revision: 47373234ac
Compiled against Qt: 5.15.3
Running against Qt: 5.15.3
Compiled against GDAL: 3.8.3
Running against GDAL: 3.8.3

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.19045

**Comment by uclaros** [comment]
This looks like the kind of crash that is getting fixed with #56320 which will be available in the new LTR next week!

**Comment by aritchie-usgs** [comment]
Appears to be resolved in current version.
